site/_data/releases.yaml

```diff
@@ -1,7 +1,7 @@
 stable:
-  name: Hammer-4
-  tag: hammer-4
-  filename: hammer-4
+  name: Hammer-5
+  tag: hammer-5
+  filename: hammer-5
```

Comment:
This part is what I was talking about earlier... I think `name` can stay as `hammer-5`, but `tag` and `filename` should be `hammer-5.1`. Can you run this locally using `miq` and make sure all look good?